common/persistence/src/main/resources/db/specific/postgresql/V4__createPermissionsForCallback.sql

```diff
@@ -1 +1 @@
-GRANT SELECT, INSERT ON TABLE federation_batch TO "cwa_federation_callback";
+GRANT SELECT, INSERT ON TABLE federation_batch_info TO "cwa_federation_callback";
```

Comment:
This migration was already applied in dev. Changing it means that we would need to reset the DB in dev manually. I guess it's not a big deal since it's only dev, but we should keep this in mind when merging this PR.